frontend/appflowy_flutter/lib/workspace/presentation/home/menu/menu.dart

```diff
@@ -104,6 +108,56 @@ class HomeMenu extends StatelessWidget {
     );
   }
 
+  Widget _renderFavorites(BuildContext context) {
+    return BlocProvider(
+      create: (context) =>
+          getIt<FavoriteBloc>()..add(const FavoriteEvent.initial()),
+      child: BlocConsumer<FavoriteBloc, FavoriteState>(
+        listener: (context, state) {
+          Log.warn("Favorite state changed $state");
+        },
```

Comment:
Once ready for review/merge, if the listener isn't being used, then prefer a BlocBuilder over Consumer. Maybe add a TODO for yourself to not forget :)